pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -728,6 +732,23 @@ func (c *ClusterConfig) AppendAvailabilityZone(newAZ string) {
 	c.AvailabilityZones = append(c.AvailabilityZones, newAZ)
 }
 
+// SetClusterStatus populates ClusterStatus using *eks.Cluster.
+func (c *ClusterConfig) SetClusterStatus(cluster *eks.Cluster) error {
+	if networkConfig := cluster.KubernetesNetworkConfig; networkConfig != nil && networkConfig.ServiceIpv4Cidr != nil {
+		c.Status.KubernetesNetworkConfig = &KubernetesNetworkConfig{
```

Comment:
are we adding this to the status for just "display purposes"? i can't see this status field being used anywhere, have i missed it

Reply:
It's used in `getClusterDNS`.